mcapi/crypto.h

```diff
@@ -167,7 +167,7 @@ enum {
 
 /* AES */
 typedef struct CRYPT_AES_CTX {
-    int holder[90];   /* big enough to hold internal, but check on init */
+    int holder[602];   /* big enough to hold internal, but check on init */
```

Comment:
This is almost a factor of 7 size increase.  Why is this needed for microchip APIs?

Reply:
For some reason I had MS CAPI in my mind.
Change the AES-GCM default for MCAPI to be the non-table version and restored the size.